File: day01/part01.py
import os

with open("day01/input.txt") as input_file:
    input = [int(line) for line in input_file.readlines()]
    for (i, n) in enumerate(input):
        for m in input[i:]:
            if n + m == 2020:
                print(n * m)

File: day01/part01.js
const fs = require("fs");

const input = fs
  .readFileSync("day01/input.txt", "utf-8")
  .trim()
  .split("\n")
  .map(Number);
for (const [i, n] of input.entries()) {
  for (const m of input.slice(i + 1)) {
    if (n + m === 2020) {
      console.log(n * m);
    }
  }
}
